diff -Naurp subscription-manager-1.28.28/etc-conf/plugin/subscription-manager.conf subscription-manager-1.28.28-patch/etc-conf/plugin/subscription-manager.conf
--- subscription-manager-1.28.28/etc-conf/plugin/subscription-manager.conf	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/etc-conf/plugin/subscription-manager.conf	2022-04-02 17:16:08.219528948 -0700
@@ -1,5 +1,5 @@
 [main]
-enabled=1
+enabled=0
 
 # When following option is set to 1, then all repositories defined outside redhat.repo will be disabled
 # every time subscription-manager plugin is triggered by dnf or yum
diff -Naurp subscription-manager-1.28.28/etc-conf/subscription-manager-cockpit.desktop.in subscription-manager-1.28.28-patch/etc-conf/subscription-manager-cockpit.desktop.in
--- subscription-manager-1.28.28/etc-conf/subscription-manager-cockpit.desktop.in	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/etc-conf/subscription-manager-cockpit.desktop.in	2022-04-02 16:57:54.295374442 -0700
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=Red Hat Subscription Manager
+_Name=Subscription Manager
 Icon=subscription-manager
 Type=Application
 TryExec=/usr/libexec/cockpit-desktop
diff -Naurp subscription-manager-1.28.28/etc-conf/subscription-manager-gui.desktop.in subscription-manager-1.28.28-patch/etc-conf/subscription-manager-gui.desktop.in
--- subscription-manager-1.28.28/etc-conf/subscription-manager-gui.desktop.in	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/etc-conf/subscription-manager-gui.desktop.in	2022-04-02 16:57:43.575412524 -0700
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=Red Hat Subscription Manager
+_Name=Subscription Manager
 Icon=subscription-manager
 Exec=subscription-manager-gui
 Terminal=false
diff -Naurp subscription-manager-1.28.28/src/subscription_manager/migrate/migrate.py subscription-manager-1.28.28-patch/src/subscription_manager/migrate/migrate.py
--- subscription-manager-1.28.28/src/subscription_manager/migrate/migrate.py	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/src/subscription_manager/migrate/migrate.py	2022-04-02 16:58:22.311274917 -0700
@@ -278,8 +278,7 @@ class MigrationEngine(object):
             if self.options.five_to_six:
                 msgs = [_("This system appears to already be registered to Satellite 6.")]
             else:
-                msgs = [_("This system appears to already be registered to Red Hat Subscription Management.")]
-                msgs.append(_("Please visit https://access.redhat.com/management/consumers/%s to view the profile details.") % identity.uuid)
+                msgs = [_("This system appears to already be registered to Subscription Management.")]
             system_exit(1, msgs)
 
         try:
diff -Naurp subscription-manager-1.28.28/src/subscription_manager/repofile.py subscription-manager-1.28.28-patch/src/subscription_manager/repofile.py
--- subscription-manager-1.28.28/src/subscription_manager/repofile.py	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/src/subscription_manager/repofile.py	2022-04-02 16:59:27.519043271 -0700
@@ -368,11 +368,7 @@ class RepoFileBase(object):
         Try to create new repo file.
         :return: None
         """
-        self.create_dir_path()
-        if self.path_exists(self.path) or not self.manage_repos:
-            return
-        with open(self.path, 'w') as f:
-            f.write(self.REPOFILE_HEADER)
+        pass
 
     def fix_content(self, content):
         return content
diff -Naurp subscription-manager-1.28.28/src/subscription_manager/repolib.py subscription-manager-1.28.28-patch/src/subscription_manager/repolib.py
--- subscription-manager-1.28.28/src/subscription_manager/repolib.py	2022-02-15 08:23:26.000000000 -0700
+++ subscription-manager-1.28.28-patch/src/subscription_manager/repolib.py	2022-04-02 16:59:07.055115968 -0700
@@ -87,7 +87,7 @@ class YumPluginManager(object):
     @staticmethod
     def warning_message(enabled_yum_plugins):
         message = _('The yum/dnf plugins: %s were automatically enabled for the benefit of '
-                    'Red Hat Subscription Management. If not desired, use '
+                    'Subscription Management. If not desired, use '
                     '"subscription-manager config --rhsm.auto_enable_yum_plugins=0" to '
                     'block this behavior.') % ', '.join(enabled_yum_plugins)
         return message
